db/revamp_cleanup.sql

```diff
@@ -0,0 +1,13 @@
+DELETE FROM node_quants a
```

Comment:
this is more of a 1-time script than something to keep around. Shall we at least flag it as such?